doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -6,7 +6,7 @@ Python Types and C-Structures
 
 Several new types are defined in the C-code. Most of these are
 accessible from Python, but a few are not exposed due to their limited
-use. Every new Python type has an associated :c:type:`PyObject *` with an
+use. Every new Python type has an associated :c:type:`PyObject *<PyObject>` with an
```

Comment:
> However, I think this whole section is somewhat lacking

I thought this passage is about the difference between `PyLongType` and `PyLongObject`. So to say every python type (long type) has an associated object (long object). I don't quite understand what you mean about every type has a pointer to an associated object?

But I tried to only fix broken links and incorrect attempts to link. Not okay?